content/en/about/why-falco.md

```diff
@@ -199,7 +199,7 @@ Falco only monitors relevant events by using filters and rules to define which e
 
 ##### Kernel-level instrumentation to capture system events
 
-Falco uses kernel-level instrumentation to capture system events by intercepting system calls and other kernel-level events in real-time. This approach allows Falco to detect and respond to security incidents more quickly and effectively, as it can analyze events before they are processed by the operating system or applications. Additionally, kernel-level instrumentation ensures that Falco is highly performant, as it can intercept and analyze events with minimal overhead.
+Falco uses kernel-level instrumentation to capture system events by intercepting system calls and other kernel-level events in real-time. This approach allows Falco to detect security incidents more quickly and effectively, as it can analyze events before they are processed by the operating system or applications. Additionally, kernel-level instrumentation ensures that Falco is highly performant, as it can intercept and analyze events with minimal overhead.
```

Comment:
Hi! I looked at this part in the context in which it's written. I believe this paragraph should be focused on the kernel component, like Aizhamal mentioned, and also avoid misconceptions related to interference with system calls but at the same time I don't think we should mention SIEMs or data lakes here as we're talking mostly about kernel instrumentation. So, here's my suggestion:

```
Falco uses kernel instrumentation to capture system events by intercepting system calls and other kernel-level signals. The Falco kernel components are designed to be fast and non intrusive, as they do not alter the system's behavior. By using this approach, Falco can collect rich information of what applications are doing nearly in real-time while minimizing overhead and preventing interference with regular workload behavior.
```

Reply:
I see that @leogr has responded at the same time as I did and couldn't read his response :sweat_smile: 
Anyways, I like the idea of using observing and monitoring as terms since this is what Falco does so I changed the suggestion accordingly.

Reply:
Love the rewrite and thanks for checking on the context here! @aijamalnk we should be good to go with this one! Cheers @LucaGuerra!